modyn/tests/trainer_server/internal/trainer/test_pytorch_trainer.py

```diff
@@ -654,10 +654,6 @@ def test_train(
     assert test_send_metadata.call_count == len(trainer._callbacks)
     test_cleanup.assert_called_once()
 
-    if not platform.system() == "Darwin":
-        assert status_queue.qsize() == 1
-    else:
-        assert not status_queue.empty()
```

Comment:
why did you remove this?

Reply:
The status_queue might not contain the result yet. We have to wait (which is done in lines 662-674).